docs/content/usage/configurations.md

```diff
@@ -80,6 +80,12 @@ DATABASE_URL=<YOUR_DATABASE_URL>
 
 Where `<YOUR_DATABASE_URL>` is the URL of your PostgreSQL database.
 
+Remember to run migrations after you change your database type to `postgres` from another type:
+
+```bash
+medusa migrations run
+```
+
```

Comment:
Can you add a new H3 header before this called "Changing Database Type" to make it easier to find?

Reply:
@shahednasser Here you go, I've added the H3 header.